.github/workflows/gke.yaml

```diff
@@ -1,29 +1,46 @@
-name: ConformanceGKE
+name: Trigger GKE connectivity test (see ConformanceGKE for results)
 
 on:
   pull_request_target:
     types:
       - "labeled"
-  #pull_request: #uncomment for testing
+  # WARNING FOR REVIEWERS
+  # The `pull_request` webhook event should never be uncommented on `master`,
+  # so as not to allow access to or leak repo's secrets.
+  # Please be extra careful when reviewing changes to GitHub workflows.
+  # pull_request:
   #  types:
   #    - "labeled"
```

Comment:
I don't understand this comment, can you clarify it a little bit further?

Reply:
This was poorly worded but the initial intention was to leave it there for workflow developers working directly on Cilium. On second thought it does not make much sense to have it no matter the case, so I've removed it.